CHANGELOG.md

```diff
@@ -5,30 +5,30 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 ## [Unreleased]
 
-### Breaking changes
+### Breaking Changes
 
-#### exonum
+#### Exonum
```

Comment:
It was intentional afaik, but I don't think it's worse now.

Reply:
It was indeed intentional. 🙃 

Subsections are named the same as our "components". Otherwise we will have to rename `exonum-testkit`, `exonum-configuration`, etc. Personally I prefer consistent naming.